src/components/v2/V2Create/FormItemLabel.tsx

```diff
@@ -1,11 +1,18 @@
 import { ThemeContext } from 'contexts/themeContext'
-import { PropsWithChildren, useContext } from 'react'
+import { ChildElems } from 'models/child-elems'
+import { CSSProperties, useContext } from 'react'
 
-export default function FormItemLabel({ children }: PropsWithChildren<{}>) {
+export default function FormItemLabel({
+  children,
+  style,
+}: {
+  children: ChildElems | string
```

Comment:
**issue**: we can keep PropsWithChildren like this:

`PropsWithChildren<{ style?: CSSProperties }>`

Reply:
Oh ok nice